ena_upload/ena_upload.py

```diff
@@ -578,7 +578,7 @@ def process_args():
 
     parser.add_argument('--tool_version',
                         dest='tool_version',
-                        default='0.3.0',
+                        default='0.3.1',
```

Comment:
we should probably make this a variable that we can reuse and only bump in one place

Reply:
I agree, using maybe a _version.py file with `__version__ = "0.3.2"` in it?

Reply:
jepp! :)

Reply:
I added it in #16 !